account_payment_order/wizard/account_payment_line_create.py

```diff
@@ -39,8 +40,14 @@ class AccountPaymentLineCreate(models.TransientModel):
         selection=[("same", "Same"), ("same_or_null", "Same or Empty"), ("any", "Any")],
     )
     move_line_ids = fields.Many2many(
-        comodel_name="account.move.line", string="Move Lines"
+        comodel_name="account.move.line",
+        string="Move Lines",
+        domain="[('id', 'in', selectable_aml_ids)]",
```

Comment:
This is totally anti-perfomant when this list gets bigger. Please use another approach using an "absolute domain".

Reply:
imho https://github.com/OCA/bank-payment/pull/1106/commits/0fc94a34651cf888f11df459833749a4dad6c564 fixes this potential performance issue.

Reply:
I don't think so. The problem is that the web client has to download a very large string containing the whole IDs list, which takes a lot of time whilst this list gets bigger. Try this PR in a DB with let's say 5.000 AR/AP pending moves to see what I'm talking about.